WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginBaseFormFragment.java

```diff
@@ -0,0 +1,195 @@
+        mSecondaryButton = (Button) rootView.findViewById(R.id.secondary_button);
+        setupBottomButtons(mSecondaryButton, mPrimaryButton);
+
+        return rootView;
+    }
+
+    @Override
+    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        Toolbar toolbar = (Toolbar) view.findViewById(R.id.toolbar);
+        ((AppCompatActivity) getActivity()).setSupportActionBar(toolbar);
+
+        ActionBar actionBar = ((AppCompatActivity) getActivity()).getSupportActionBar();
+        if (actionBar != null) {
+            actionBar.setDisplayShowTitleEnabled(false);
+            actionBar.setDisplayHomeAsUpEnabled(true);
+        }
+
+        if (savedInstanceState == null) {
+            EditTextUtils.showSoftInput(getEditTextToFocusOnStart());
+        }
+    }
+
+    @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+
+        if (savedInstanceState != null) {
+            mInProgress = savedInstanceState.getBoolean(KEY_IN_PROGRESS);
+
+            if (mInProgress) {
+                showProgressDialog();
+            }
+        }
+    }
+
+    @Override
+    public void onAttach(Context context) {
+        super.onAttach(context);
+        if (context instanceof LoginListener) {
+            mLoginListener = (LoginListener) context;
+        } else {
+            throw new RuntimeException(context.toString() + " must implement LoginListener");
+        }
+    }
+
+    @Override
+    public void onDetach() {
+        super.onDetach();
+        mLoginListener = null;
+    }
+
+    @Override
+    public void onStart() {
+        super.onStart();
+        mDispatcher.register(this);
+    }
+
+    @Override
+    public void onStop() {
+        super.onStop();
+        mDispatcher.unregister(this);
+    }
+
+    @Override
+    public void onSaveInstanceState(Bundle outState) {
+        super.onSaveInstanceState(outState);
+
+        outState.putBoolean(KEY_IN_PROGRESS, mInProgress);
+    }
+
+    @Override
+    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+        inflater.inflate(R.menu.menu_login, menu);
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        if (item.getItemId() == R.id.help) {
+            mLoginListener.help();
+            return true;
+        }
+
+        return false;
+    }
+
+    protected void showProgressDialog() {
+        mPrimaryButton.setEnabled(false);
+        mSecondaryButton.setEnabled(false);
+
+        mProgressDialog =
+                ProgressDialog.show(getActivity(), "", getActivity().getString(getProgressBarText()), true, true,
+                        new DialogInterface.OnCancelListener() {
+                            @Override
+                            public void onCancel(DialogInterface dialogInterface) {
+                                if (mInProgress) {
```

Comment:
The `if (mInProgress)` statement was changed to `if (isInProgress())` in 33673d5b6361df192fdb8b739fffd1add4ae6d71 so the lint warning is no longer visible.  I'm fine with leaving `isInProgress()` if you want.